Support `expires_in` for `List`, `Set`, `UniqueList`, `OrderedSet` (#143)
rails/kredis: README.md

false == limiter.exceeded?      # => GET "limiter"
```

Lists, unique lists, sets, ordered sets, and hashes support expiration:

```ruby
set = Kredis.set "myset", expires_in: 1.second
set.add "hello", "world" # => SADD myset "hello" "world"
true == set.include?("hello") # => SISMEMBER myset "hello
sleep 2
[] == set.members # => SMEMBERS myset
```

> To support lower versions of redis, which does not has `nx` option on `EXPIRE` command, multiple commands are used to achieve the same effect.

Reviewer: ```suggestion
To support lower versions of Redis which do not have the `nx` option on `EXPIRE`, multiple commands are used to achieve the same effect.
```
